packages/trader/src/App/app.tsx

```diff
@@ -13,7 +13,7 @@ import TraderProviders from '../trader-providers';
 type Apptypes = {
     passthrough: {
         root_store: TCoreStores;
-        WS: unknown;
+        WS: Record<string, unknown>;
```

Comment:
WS is not a record of strings. It should be the type in the image. This type is in bots package. Can we move it and use it here too?
<img width="911" alt="image" src="https://github.com/kate-deriv/deriv-app/assets/56330681/21b8e981-375c-4d03-96d6-d6a075a10df1">

Reply:
@balakrishna-deriv yes, i will use it then, and will add WS methods that we use in `trader` package